README.md

```diff
@@ -213,5 +213,5 @@ functions-framework --help
 
 ## For more information
 
-See the examples directory for additional examples, and consult the gem
-documentation on rubydoc.info for full reference documentation.
+See the examples directory for additional examples, and consult
```

Comment:
It seems like this sentence is pointing out two separate pieces of information, samples and reference docs. Separate them with two sentences or two bullet points for quicker skimming?

Reply:
Okay